# HG changeset patch
# User rk
# Date 1286206634 -7200
# Node ID e04a2f496a99955944084b847af1a762358c5b9e
# Parent  0f65d0d2d8f821192283646fc8ebee566a866940
reset m_underMouse in wxDataViewMainWindow::Scroll[Window/To] methods

diff -r 0f65d0d2d8f8 -r e04a2f496a99 src/generic/datavgen.cpp
--- a/src/generic/datavgen.cpp	Mon Oct 04 17:31:43 2010 +0200
+++ b/src/generic/datavgen.cpp	Mon Oct 04 17:37:14 2010 +0200
@@ -2160,6 +2160,8 @@ void wxDataViewMainWindow::RecalculateDi
 
 void wxDataViewMainWindow::ScrollWindow( int dx, int dy, const wxRect *rect )
 {
+	m_underMouse = NULL;
+
     wxWindow::ScrollWindow( dx, dy, rect );
 
     if (GetOwner()->m_headerArea)
@@ -2168,6 +2170,8 @@ void wxDataViewMainWindow::ScrollWindow(
 
 void wxDataViewMainWindow::ScrollTo( int rows, int column )
 {
+	m_underMouse = NULL;
+
     int x, y;
     m_owner->GetScrollPixelsPerUnit( &x, &y );
     int sy = GetLineStart( rows )/y;
